core/src/main/scala/cats/syntax/either.scala

```diff
@@ -306,6 +308,14 @@ final class RightOps[A, B](val right: Right[A, B]) extends AnyVal {
   def leftCast[C]: Either[C, B] = right.asInstanceOf[Either[C, B]]
 }
 
+final class EitherIdOps[A](val obj: A) extends AnyVal {
```

Comment:
@kailuowang I have a PR (#1456) making the ops classes consistent in this regard.